### PR TITLE
Add Travis CI

### PR DESCRIPTION
**Pull request by RikkiGibson**

### CODE REVIEW

**Comment by RikkiGibson** [comment]
Welp, whitespace strikes again

**Comment by RikkiGibson** [comment]
:tada:

**Comment by RikkiGibson** [comment]
50% of the time, it works every time @jschneidereit

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: csharp
```
Comment (jschneidereit) [inline .travis.yml:1]: pretty slick!

File: MsTestToXunitConverter.xUnit/Extensions/SyntaxNodeExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        /// <summary>
+        /// Converts a syntax node to a string, ensuring that the appropriate
+        /// new line characters for the current environment are used.
+        /// </summary>
+        internal static string ToEnvNewlineString(this SyntaxNode node)
+        {
+            var content = node.ToFullString();
+            if (Environment.NewLine == "\n")
+            {
+                content = content.Replace("\r\n", "\n");
```
Comment (jschneidereit) [inline MsTestToXunitConverter.xUnit/Extensions/SyntaxNodeExtensions.cs:21]: Gosh darn cross platform

File: MsTestToXunitConverter/AssertRewriter.cs [hunk 3]
```diff
@@ -28,6 +28,7 @@ private static InvocationExpressionSyntax InvocationRewriter(this InvocationExpr
             {
                 if (identifier != null)
                 {
+                    identifier = identifier.WithLeadingTrivia(mae.Expression.GetLeadingTrivia());
```
Comment (jschneidereit) [inline MsTestToXunitConverter/AssertRewriter.cs:31]: :blush:

**Review by jschneidereit — APPROVED** [review]
Looks good!

**Pull request merged**